app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -545,14 +204,6 @@ define([
         }
     }
 
-    function getBootDoneChart(response) {
-        chart.bootpie({
-            element: 'boot-chart',
-            response: response,
-            legend: true
-        });
-    }
-
     function getBoots() {
         var deferred;
 
```

Comment:
These boot results are not needed, and in fact they're not used anywhere.  You should just have to call `getPlans`.